test/fixtures/dialects/vertica/utf8.sql

```diff
@@ -0,0 +1,4 @@
+--https://docs.vertica.com/24.3.x/en/sql-reference/language-elements/identifiers/
```

Comment:
This is quite a fundamental change, looks good though. Woudl you mind adding more test cases?